### DISCUSSION TITLE
Build Performance across all Heroes visualized in one place

### DISCUSSION THREAD

**Discussion by Zemill** [Heroes-Profile/heroesprofile#1000] [Ideas]
Performance of different builds, in say, Master rank. You might say, that's already a feature on Heroes Profile! But more specifically, I mean out of context of a particular hero. So I could sort out of all builds for all heroes, to discover the highest winrate builds in Master. Then I could look at a top winrate build that looks appealing, and decide I want to master that build for that hero! Even better to see it by map, to know what build on which hero performs really well on the map. Maybe it's a fun idea that could be implemented using the data already available!

**Comment by Frankpro42** [comment]
It's a good idea and I'd like to see the playstyle (replays) related to the build which I find as much important as the build by itself especially from OTPs.

**Comment by HenrikRaz** [comment]
I would also find this interesting. Calculating some kind of popularity-weighted winrate would be crucial though, as you otherwise just get builds with almost no games played in the top.

**Reply by Hunter-Bobeck** [reply]
Agreed, such as having a cutoff to display only those builds with X+ number of games played
